database/sqlite/sqlite_health.c

```diff
@@ -1345,7 +1345,7 @@ void sql_health_alarm_log2json(RRDHOST *host, BUFFER *wb, uint32_t after, char *
         if (sqlite3_column_type(res, 31) != SQLITE_NULL)
             uuid_unparse_lower(*((uuid_t *) sqlite3_column_blob(res, 31)), transition_id);
 
-        char *edit_command = health_edit_command_from_source((char *)sqlite3_column_text(res, 17));
+        char *edit_command = sqlite3_column_bytes(res, 17) > 0 ? health_edit_command_from_source((char *)sqlite3_column_text(res, 17)) : strdupz("UNKNOWN=0=UNKNOWN");
```

Comment:
Hello @MrZammler ,

Considering that you are using a `strdupz` in a function that you won't return the value, we should use `freez` before the function ends.

Best regards!

Reply:
Hi Thiago! There is a `freez(edit_command);` a few lines below (1457), or did I not understand correctly?

Reply:
I am sorry, I missed that :+1: